src/app/portlets/content-types/content-types-resolver.service.ts

```diff
@@ -42,6 +42,10 @@ export class ContentTypeResolver implements Resolve<ContentType> {
                     this.dotRouterService.gotoPortlet('/content-types-angular');
                     return null;
                 }
+            })
+            .catch(() => {
+                this.dotRouterService.gotoPortlet('/content-types-angular');
```

Comment:
here, use as a second parameter, 'true'. so the URL is replaced and don't keep the invalid entry in the browser history

Reply:
Good catch!